src/overlay/components/RuntimeErrorStack.js

```diff
@@ -28,7 +28,14 @@ function RuntimeErrorStack(document, root, props) {
   stackContainer.style.lineHeight = '1.3';
   stackContainer.style.whiteSpace = 'pre-wrap';
 
-  const errorStacks = ErrorStackParser.parse(props.error);
+  let errorStacks;
+  try {
+    errorStacks = ErrorStackParser.parse(props.error);
+  } catch (e) {
+    errorStacks = [];
+    stackContainer.innerHTML = 'No stack trace';
```

Comment:
might be good to put a JSON dump of props.error as well as e.message for better debug-ability?

Reply:
The expected exception is `throw new Error('Cannot parse given Error object');
 ` from https://github.com/stacktracejs/error-stack-parser/blob/master/error-stack-parser.js#L35, which doesn't look particularly useful.  It's also not something an end-user would want to debug -- the error would mainly be useful to authors of this plugin.

Reply:
got it, fair enough! (I'm not an approver, just looking at pull requests for fun)

Reply:
I think 'No stack trace is available for this Error!' seems clearer?

Otherwise I think this is good to merge.

Reply:
Updated the message